xonsh/ptk/completer.py

```diff
@@ -2,6 +2,7 @@
 """Completer implementation to use with prompt_toolkit."""
 import os
 import builtins
+import xonsh
```

Comment:
It's working this way but maybe it's a bad practice so I'll change to `xonsh.shell` sorry for waisting your time